custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -222,18 +222,13 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         return locationsService.getLocations(level, locationsCache, vm.selectedLocations, vm.disallowNational());
     };
 
-    var selectedLocationIndex = function () {
```

Comment:
This is just a minor refactor to reuse the function defined in location service.